test/Microsoft.Azure.WebJobs.Host.UnitTests/Loggers/ApplicationInsightsConfigurationTests.cs

```diff
@@ -679,6 +700,46 @@ public void DependencyInjectionConfiguration_ConfiguresActiveOnlyOnce()
             }
         }
 
+        [Fact]
+        public void CreateFilterOptions_EnableLiveMetricsFilters()
+        {
+            using (var host = new HostBuilder()
+                .ConfigureLogging(b =>
+                {
+                    b.AddApplicationInsightsWebJobs(o =>
+                    {
+                        o.InstrumentationKey = "some key";
+                        o.EnableLiveMetricsFilters = true;
+                    });
+                })
+                .Build())
+            {
+                var filterOptions = host.Services.GetService<IOptions<LoggerFilterOptions>>().Value;
+                var rule = SelectAppInsightsRule(filterOptions, "Category");
+                Assert.Equal(null, rule.Filter);
```

Comment:
nit: you can use `Assert.IsNull()` for these. No need to update this, but just for future reference :-)